rpa-manager/modules/ROOT/pages/alerting-rule.adoc

```diff
@@ -1,6 +1,6 @@
 = Defining Rules for Alerts
 
-Specify the incidents that will trigger an alert. Each activated rule will be checked while configurations are deployed.
+Specify the incidents that trigger an alert. Each activated rule be checked while configurations are deployed.
```

Comment:
```suggestion
Specify the incidents that trigger an alert. Each activated rule is checked while configurations are deployed.
```

Reply:
We try not to use the future tense, but in the text above, it sounds strange to use the present tense.